data/outreachy.yml

```diff
@@ -276,4 +276,19 @@ rounds:
         - Reynir Björnsson
         video: https://watch.ocaml.org/w/kTwq4AVQtRtFuQx9R5cPro
 
-       
+  - name: Summer 2023
+    projects:
+      - title: "MIDI over Ethernet with MirageOS"
+        description: >
+          MIDI, which stands for Musical Instrument Digital Interface, is a widely used protocol in the world of music and audio technology.
+          MirageOS is a library operating system that specializes in creating lightweight, secure, and efficient unikernels. Unikernels are highly specialized, 
+          single-purpose virtual machine images designed for specific applications, and is writting in OCaml.
+          The project focused on implementing the rtpMIDI protocol for serializing-deserializing of MIDI messages over Ethernet. And implementing usecases like
+          a publisher-subcriber based server-client model for MIDI messages.
+        mentee: "Aryan Godara"
+        source: https://github.com/AryanGodara/rtpmidi
+        mentors:
+        - rand
```

Comment:
Would people know who this is?

Reply:
I'll add rand's legal name 😅 
But this is what was written on the Outreachy website.

Reply:
I'm still working on this :)

Reply:
No problem, just let us know when to review. It helps to make it a draft pull request to clarify, if necessary.

Reply:
@sabine I've updated the name. So the PR can be reviewed :)
I'm really sorry for the delay

Reply:
All good, no worries. :slightly_smiling_face: